### PR TITLE
Issue 2840: Hyperlinks added to hdfs.yml and zookeeper.yml

### PR DESCRIPTION
**Pull request by arvindkandhare**
Signed-off-by: arvindkandhare <arvind.kandhare@emc.com>

**Change log description**  
Hyperlinks added to hdfs.yml and zookeeper.yml in docker-swarm documentation

**Purpose of the change**  
 Fixes #2840

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -49,7 +49,7 @@ typesafeConfigVersion=1.3.1
 gradleGitPluginVersion=2.2.0
 
 # Version and base tags can be overridden at build time
-pravegaVersion=0.4.0-SNAPSHOT
+pravegaVersion=0.3.1-SNAPSHOT
```
Comment (andreipaduroiu) [inline gradle.properties]: This doesn't look right. Why are we going back?

File: documentation/src/docs/streamcuts.md [hunk 2]
```diff
@@ -92,4 +92,4 @@ It must be noted that passing ```StreamCut.UNBOUNDED``` to startStreamCut and en
 current head of stream and the current tail of the stream, respectively.
 
 
-We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/v0.3.0/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
+We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/v0.3.1/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
```
Comment (andreipaduroiu) [inline documentation/src/docs/streamcuts.md]: There are a number of comments on this line in #2850 . Let's follow up here.
https://github.com/pravega/pravega/pull/2850#discussion_r211338727

File: gradle.properties (hunk 1 above)
Comment (fpj) [reply gradle.properties]: It looks like an accidental change, this is clearly incorrect.

**Comment by Aparnarr** [comment]
losing this PR in favour of #2930

**Pull request closed without merging**